src/components/dashjs-player/dashjs-player.tsx

```diff
@@ -110,15 +137,93 @@ export class DashjsPlayer {
     });
   }
 
+  /* Stream Metrics */
   @Event()
-  streamMetricsEvent: EventEmitter<any>;
+  metricsEvent: EventEmitter<string>;
+
+  private metricsWatch() {
+    this.player && this.metricsEvent.emit(this.streamMetrics(this.player, this.metrics));
+  }
+
+  private streamMetrics(player: any, metrics: any) {
```

Comment:
can you type the functions? (should be done either way as it should be reported as lint error)

Reply:
document them you mean?

Reply:
No, don't use any. As any can be literally anything